core/tests/protractor_utils/action.js

```diff
@@ -69,10 +69,15 @@ var select2 = async function(selectorName, selectorElement, optionToSelect) {
 };
 
 var sendKeys = async function(
-    inputName, inputElement, keys, clickInputElement = true) {
+    inputName, inputElement, keys, isRte = false, clickInputElement = true) {
   if (clickInputElement) {
     await click(inputName, inputElement);
   }
+  if (isRte) {
+    var rteToolbar = element(by.id('cke_editor1'));
```

Comment:
Just an FYI, I think this would fail when there are multiple instances of ck editor on the same page and the input does not correspond to "cke_editor1".